encoder/vaapiencoder_h264.cpp

```diff
@@ -700,13 +700,14 @@ class VaapiEncoderH264Ref
     SurfacePtr m_pic;
 };
 
-VaapiEncoderH264::VaapiEncoderH264():
-    m_numBFrames(0),
-    m_reorderState(VAAPI_ENC_REORD_WAIT_FRAMES),
-    m_streamFormat(AVC_STREAM_FORMAT_ANNEXB),
-    m_frameIndex(0),
-    m_keyPeriod(30),
-    m_idrNum(0)
+VaapiEncoderH264::VaapiEncoderH264()
```

Comment:
Next time, please do not mix style change commit with the real change. This you can see why https://wiki.qt.io/Commit_Policy  in 8.1